code/game/objects/items/tools/misc_tools.dm

```diff
@@ -12,43 +12,34 @@
 	icon_state = "labeler0"
 	item_state = "flight"
 	var/label = null
-	var/labels_left = 50
 	var/mode = 0	//off or on.
-
+	var/wait = 0
+	var/labels = 50
 /obj/item/tool/hand_labeler/afterattack(atom/A, mob/user as mob, proximity)
 	if(!proximity) return
 	if(!mode)	//if it's off, give up.
 		return
 	if(A == loc)	// if placing the labeller into something (e.g. backpack)
 		return		// don't set a label
-
-	if(!labels_left)
-		to_chat(user, "<span class='notice'>No labels left.</span>")
-		return
 	if(!label || !length(label))
 		to_chat(user, "<span class='notice'>No text set.</span>")
 		return
 	if(length(A.name) + length(label) > 64)
 		to_chat(user, "<span class='notice'>Label too big.</span>")
 		return
-	if(isliving(A))
```

Comment:
Do we want to be able to label people? :thinking:

Reply:
No we don't, or really any mob.